### DISCUSSION TITLE
trouble in the Connection with srsgnb and the Samsung S21 FE 5G

### DISCUSSION THREAD

**Discussion by alyfekry12** [srsran/srsRAN_Project#563] [srsRAN-matlab]
We are trying to connect Samsung S21 FE 5G to srsGnb but the network doesn't appear in the network selection Is there any configuration to do in the Tx or Rx section ? Or If is anyone tried this mobile please Help.

**Comment by pgawlowicz** [comment]
seems there are some issues with Samsung S21, please check here: https://github.com/srsran/srsRAN_Project/discussions/36#discussioncomment-8818537

**Reply by pgawlowicz** [reply]
@alyfekry12 any update on this issue?

**Reply by alyfekry12** [reply]
Still trying to see the network but is there any rf configuration for this device to do ?(Note: I am also trying to connect to the OAI Gnb)

**Reply by improperaffo** [reply]
Hi @alyfekry12, which PLMN are you using? And are you forcing the device to be in 5G only?

**Reply by alyfekry12** [reply]
I am using 208 93 and I forced the device to 5G using 5G switch app and also I have rooted the mobile and used network signal guru to force 5G through but I can't enter the advanced mode in the app so any help ?

**Reply by improperaffo** [reply]
We also encountered some issues when using PLMN 20893, I would suggest you try again with PLMN 00101
